### PR TITLE
Have bot accrue a count of morning points when a user says GOOD MORNING

### PR DESCRIPTION
**Pull request by crscillitoe**

### CODE REVIEW

File: db/point_accrual.py [hunk 1]
```diff
@@ -28,6 +29,69 @@ def get_multiplier_for_user(roles: list[Role]) -> int:
...
+    with session() as sess:
+        result = sess.execute(
+            select(MorningPoints).where(MorningPoints.user_id == user_id)
+        ).first()
+
+        if result is None:
+            return 0
+
+        morning_points: MorningPoints = result[0]
+        return morning_points.weekly_count
```
Comment (teknogeek) [inline db/point_accrual.py:94]: Since this query is going to be run fairly often, can we outdent this so the session closes ASAP?

**Review by teknogeek — CHANGES_REQUESTED** [review]
LGTM otherwise

File: db/models.py [hunk 2]
```diff
@@ -99,6 +99,14 @@ class RoleModifier(Base):
     def __repr__(self):
         return f"RoleModifier(id={self.id!r}, guild_id={self.guild_id!r}, role_id={self.role_id!r}, modifier={self.modifier!r})"
 
+class MorningPoints(Base):
+    __tablename__ = "morning_points"
+    user_id = Column(BigInteger, primary_key=True)
+    timestamp = Column(DateTime, default=func.now(), onupdate=func.now())
+    weekly_count = Column(Integer, nullable=False, default=0)
```
Comment (braddotcoffee) [inline db/models.py:106]: Can we rename this to `daily_count` since it accrues once per day rather than once per week?

**Review by braddotcoffee — CHANGES_REQUESTED** [review]
Overall looks good

File: db/models.py (hunk 2 above)
Comment (crscillitoe) [reply db/models.py:106]: it resets once a week

Comment (braddotcoffee) [reply db/models.py:106]: But it counts the number of days since last reset, right?

Comment (braddotcoffee) [reply db/models.py:106]: My main issue here is if I saw `weekly_count=2` I would expect that to mean that the user has said good morning for two consecutive weeks

**Pull request merged**